docs/src/main/asciidoc/quickstart.adoc

```diff
@@ -43,13 +43,23 @@ The mini-application's `Environment` is used to enumerate property sources and p
 The HTTP service has resources in the following form:
 
 ----
-/{application}/{profile}[/{label}]
+/{application}/[{profile}]/{label}
```

Comment:
This change needs to be reverted

Reply:
there was a comment from you saying that `[XXX]` notation means optional? If so what is this based on? I was trying to get through the openAPI documentation to find this, but no luck. I was wondering if it would make more sense to present in in the examples as:

```
/{application}/{profile}
/{application}/{profile}/{label}
```

Reply:
I have no idea what is in the openAPI documentation. On our team, `[xyz]` means optional in this case.

Reply:
I see, thank you for letting me know.